scripts/recan_bundle_tdr.py

```diff
@@ -329,18 +353,33 @@ def main(argv):
     Load a canned bundle from DCP/1 and write *.manifest.tdr and *.metadata.tdr
     files showing the desired output for DCP/2.
     """
+    default_version = datetime(year=2021,
+                               month=1,
+                               day=17,
+                               hour=0)
+
     parser = argparse.ArgumentParser(description=__doc__)
     parser.add_argument('--bundle-uuid', '-b',
-                        help='The UUID of the existing DCP/1 canned bundle.')
+                        default=TestTDRPlugin.bundle_uuid,
+                        help='The UUID of the existing DCP/1 canned bundle '
+                             '(default: %(default)s).')
```

Comment:
There's a help formattter that prints the default automatically. Some of our scripts already use that and so should this one, if possible.